pkg/keycloak/keycloak.go

```diff
@@ -52,23 +52,28 @@ func NewHandler(kcClientFactory KeycloakClientFactory, svcCatalog scclientset.In
 }
 
 func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
+	logrus.Debug("handling object ", event.Object.GetObjectKind().GroupVersionKind().String())
+
 	kc := event.Object.(*v1alpha1.Keycloak)
 	kcCopy := kc.DeepCopy()
 	namespace := kc.ObjectMeta.Namespace
 
+	logrus.Debugf("Keycloak: %v, Phase: %v", kc.Name, kc.Status.Phase)
+
+	if event.Deleted {
+		return nil
+	}
```

Comment:
This is needed as it seems to constantly send events after it's deleted if you don't. I think it's because we fail with an error further down in that scenario which causes the operator to increase the events?